spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-sample/src/main/java/com/example/WebController.java

```diff
@@ -69,22 +70,33 @@ public RedirectView publish(@RequestParam("topicName") String topicName,
 	}
 
 	@GetMapping("/pull")
-	public RedirectView pull(@RequestParam("subscription") String subscriptionName) {
```

Comment:
This is supposed to be a simple example for people to get started with. Now it's a lot more complicated. Maybe leave the original, and add another endpoint called `pullFromTwo` or something?

Reply:
Split into two methods: pull and multipull.